### PR TITLE
man: update the description of interval limiting

### LINKED ISSUE #7125: StartLimitIntervalSec docs issue
> Configure unit start rate limiting. By default, units which are started more than 5 times within 10 seconds are not permitted to start any more times until the 10 second interval ends.

https://www.freedesktop.org/software/systemd/man/systemd.unit.html#

This sounds like after 10 seconds (re)start will be retried but as far as I know this is not the case. 
It's also not immediately clear where the 5 (burst) and 10 (limit) come from.

### PR DESCRIPTION
**Pull request by keszybz**
Fixes #7125.

### CODE REVIEW

File: man/systemd.unit.xml [hunk 1]
```diff
@@ -764,12 +764,12 @@
...
-        <term><varname>StartLimitIntervalSec=</varname></term>
-        <term><varname>StartLimitBurst=</varname></term>
+        <term><varname>StartLimitIntervalSec=<replaceable>interval</replaceable></varname></term>
+        <term><varname>StartLimitBurst=<replaceable>burst</replaceable></varname></term>
 
-        <listitem><para>Configure unit start rate limiting. By default, units which are started more than 5 times
-        within 10 seconds are not permitted to start any more times until the 10 second interval ends. With these two
-        options, this rate limiting may be modified. Use <varname>StartLimitIntervalSec=</varname> to configure the
+        <listitem><para>Configure unit start rate limiting. Units which are started more than
+        <replaceable>burst</replaceable> times within <replaceable>interval</replaceable> interval
```
Comment (poettering) [inline man/systemd.unit.xml]: should this be "within a <replacable>interval</replaceable> time interval"? or so? i.e. with the extra "a" and the clarification "time interval"?

Comment (OlafvdSpek) [reply man/systemd.unit.xml]: Isn't interval the wrong term anyway? 
The interval between (two) start attempts is set by RestartSec AFAIK.
The 10s is more like a time window / period.

Comment (keszybz) [reply man/systemd.unit.xml]: Updated.

Comment (keszybz) [reply man/systemd.unit.xml]: > Isn't interval the wrong term anyway?

"Time window" and "time interval" are two interchangeable terms. Time period has different meaning, it's used more when something repeats. I don't think there's any confusion in this context, and using "interval" is appropriate became the option is called so.

**Comment by keszybz** [comment]
Updated.

**Comment by poettering** [comment]
lgtm

**Pull request merged**